### DISCUSSION TITLE
How to change default startup page

### DISCUSSION THREAD

**Discussion by dodiind** [Budibase/budibase#4677] [Help] [answered]
Hi ,
I just started to build a simple application , In the home page I just added an image and few title there, then I added new page called features page which contains some cards.
I notice when I run or publish the application the default startup page is changed from home to features page which this is not correct the default startup page should be still in the home page.
How to change default startup page ? is there any feature for that ? because I couldn't find it the application setting or layout

**Comment by mjashanks** [comment, marked as answer]
The default startup page will be `/` - make sure that this page exists from any user level that needs access.

If you don't have a `/`, I think the default page will be fairly random at the moment, so just make sure you have `/`!
